### PR TITLE
Publicize: Fix google plus icon

### LINKED ISSUE #9687: Publicize: missing Google plus icon
Problem is with republicize actions:
The google plus service icon is missing:

zrzut ekranu 2016-11-29 o 14 31 38

First noticed by @gziolo

### PR DESCRIPTION
**Pull request by artpi**
Fixes #9687 

Fixes google plus icon in publicize actions, which stopped displaying some time ago.

CC @gwwar @gziolo

### CODE REVIEW

File: client/my-sites/posts/post-share.jsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const PostSharing = React.createClass( {
 					'is-active': ( this.state.skipped.indexOf( connection.keyring_connection_ID ) === -1 )
 				} ) }
 			>
-				<SocialLogo icon={ connection.service }/>
+				<SocialLogo icon={ connection.service === 'google_plus' ? 'google-plus' : connection.service }/>
```
Comment (gziolo) [inline client/my-sites/posts/post-share.jsx]: What do you think about using [kebabCase](https://lodash.com/docs/4.17.2#kebabCase) from lodash in here? :)

**Comment by gziolo** [comment]
I can confirm that iIt fixes the issue. There is another strange behavior for a post I recently shared. When I click share icon I see the following:
screen shot 2016-11-29 at 15 34 01

Somehow only Google Plus icon is active. I'm not sure why that happened this way. I shared to all services I have connected to my site.

**Comment by gziolo** [comment]
Marking at ready to merge as it fixes the root issue :)

File: client/my-sites/posts/post-share.jsx (hunk 1 above)
Comment (gziolo) [inline client/my-sites/posts/post-share.jsx]: WFM :)

Comment (gwwar) [inline client/my-sites/posts/post-share.jsx]: 👍 Must have missed this one, there was drift between the social logos repo and the component in Calypso. See #9399

Just fyi that any logo name from social logos will be in kebab case.

**Pull request merged**